.github/workflows/build_docker.yml

```diff
@@ -20,6 +20,52 @@ on:
     types: [released]
 
 jobs:
+  focal:
+    name: Build and publish focal image
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: true
+    - name: Build image
+      uses: docker/build-push-action@v1
+      with:
+        repository: fairnle/nle
```

Comment:
This is fine, but we need to change `repository` in the bionic jobs to `nle-bionic`, otherwise they'll overwrite each other.